### PR TITLE
feat: Added a global css file, created a header.

### PR DESCRIPTION
**Pull request by Endorze**
image
Created a header that can be copy pasted into all future html files.

Created a global.css that should contain the css for everything that we reuse on more than 1 html page to reduce repeating CSS in several css files.

Edited file: Added hover effect to A-tag elements, added a hover effect to button to change color when hovered.

### CODE REVIEW

File: styles/global.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    --font-size-text: 1.2rem;
+    --color-text: white;
+    font-family: "Roboto";
+    /*To get this font on each page you need to insert this into head of every html page.
+        <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+    */
+}
+
```
Comment (maysunktong) [inline styles/global.css:17]: Do we need to add a universal selector * for box-sizing?

File: styles/global.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
+:root {
+    --color-blue: #7388A0;
+    --color-darkblue: #313640;
+    --color-lightbeige: #FAF2E6;
+    --color-darkbeige: #DAA46D;
+    --color-brown: #9C4F20;
+    --color-darkgreen: #766621;
+    --font-size-text: 1.2rem;
+    --color-text: white;
+    font-family: "Roboto";
```
Comment (maysunktong) [inline styles/global.css:10]: I think we could add an optional font to fall back on. e.g. font-family: "Roboto", Arial, sans-serif

**Review by maysunktong — APPROVED** [review]
overall it looks good to me.

File: styles/global.css (hunk 1 above)
Comment (Endorze) [reply styles/global.css:17]: I could absolutely do that, i mostly picked div specifically based on performance, but i do realise that our project will not be big enough for it to really make a difference, i'll switch to the * selector at my next PR. 😄

File: styles/global.css (hunk 2 above)
Comment (Endorze) [reply styles/global.css:10]: Oops i must've missed this, will totally add that, thanks!

File: styles/global.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    -webkit-box-shadow: 0px 1px 1px 0px var(--color-lightbeige);
+    -moz-box-shadow: 0px 1px 1px 0px var(--color-lightbeige);
+    box-shadow: 0px 1px 5px 0px var(--color-lightbeige)
+}
+
+.header-logotype {
+    position: absolute;
+    height: 100%;
+    left: 8vw;
+}
```
Comment (bosilf) [inline styles/global.css:48]: Perhaps, with the logotype change the color to one of the palette colors, either the original image or with a css filter.

you can use this website, but the colors may not be perfect:
https://isotropic.co/tool/hex-color-to-css-filter/

File: styles/global.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    text-decoration: none;
+    color: var(--color-darkblue);
+    font-weight: bold;
+    padding: 0.3rem 0;
+    transition: color 0.3s ease;
+}
+
+.header-nav-bar a:hover {
+    color: var(--color-darkbeige); 
+}
```
Comment (bosilf) [inline styles/global.css:70]: Add a cursor: pointer;

File: header.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <img class="header-logotype" src="./assets/header/capyzoo-logotype.png">
```
Comment (bosilf) [inline header.html:18]: Add a href to the homepage for when you click on the logotype.

**Review by bosilf — COMMENTED** [review]
Overall structure is good, would just add some details for an enhanced UX.

**Pull request merged**